### ISSUE TITLE
Authsign formating bug

### ISSUE THREAD

**Issue by YurkoWasHere** [starlinglab/integrity-backend#124]
Authsign json is saved in an additional json encode.
```
"{\"hash\":\"......
```

Seems that authsign data is read in as a string here
https://github.com/starlinglab/integrity-backend/blob/main/integritybackend/file_util.py#L216

But saved as if it was a dict being turning into a string
https://github.com/starlinglab/integrity-backend/blob/main/integritybackend/file_util.py#L221

To solve:

Load the string into a dict
```
authsign_proof = json.loads(r.text)
```
```
authsign_proof = r.json()
```

OR

Save it as a string
```
authsign_proof = r.text
```

**Comment by makeworld-the-better-one** [comment]
Fix merged into `dev`, @YurkoWasHere this issue can be closed once `dev` is merged into `main`.
